nntrainer/layers/rnn.cpp

```diff
@@ -47,6 +47,11 @@ void RNNLayer::finalize(InitLayerContext &context) {
   output_dim = input_dim;
   output_dim.width(unit);
 
+  if (dropout_rate > epsilon) {
+    wt_idx[RNNParams::dropout_mask] = context.requestTensor(
+      output_dim, "RNN:dropout_mask", true, ITERATION_LIFESPAN);
```

Comment:
`true` -> false`? true is for trainable tensor (which has gradient as well). Do we need gradient for mask tensor itself?

Reply:
is it? I thought it is "assign now" variable. I will fix it. Thanks.

Reply:
assign now is not part of layer_context interface. It is part of only Var_Grad and used only for internal purpose.